pmd-cataloger/src/main/java/sfdc/sfdx/scanner/pmd/PmdRuleCataloger.java

```diff
@@ -103,6 +104,11 @@ private void processCategoryFile(String language, String path) {
     // STEP 1: Turn the category file's XML into a Document object with a Root Element that we can actually use.
     Document doc = XmlReader.getInstance().getDocumentFromPath(path);
     Element root = doc.getDocumentElement();
+    // If the root node isn't of type 'ruleset', this isn't a valid category file, so we should just log something and skip it.
+    if (!root.getTagName().equalsIgnoreCase("ruleset")) {
```

Comment:
Don't want to check the namespace here to be a bit more sure?

xmlns="http://pmd.sourceforge.net/ruleset/2.0.0" or just xmlns="http://pmd.sourceforge.net*"

Reply:
Good idea. I'll add that.